apps/landingPage/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "gatsby-starter-default",
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "version": "0.1.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "dependencies": {
+    "@kiwicom/orbit-components": "^0.28.2",
+    "@kiwicom/orbit-design-tokens": "^0.3.0",
+    "gatsby": "^2.0.105",
+    "gatsby-image": "^2.0.20",
+    "gatsby-plugin-manifest": "^2.0.15",
+    "gatsby-plugin-offline": "^2.0.22",
+    "gatsby-plugin-react-helmet": "^3.0.2",
+    "gatsby-plugin-sharp": "^2.0.19",
+    "gatsby-source-filesystem": "^2.0.8",
+    "gatsby-transformer-sharp": "^2.1.13",
+    "prop-types": "^15.6.2",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
+    "react-helmet": "^5.2.0",
+    "styled-components": "^4.1.3"
```

Comment:
could you move dependencies that are already included in other `package.json` to peerDependency? like `@kiwicom/orbit-components` and `react`

Reply:
Moved in the latest commit